builder/builder-manifest.c

```diff
@@ -1846,6 +1872,8 @@ builder_manifest_create_platform (BuilderManifest *self,
       g_ptr_array_add (args, g_strdup ("--update"));
       g_ptr_array_add (args, g_strdup ("--writable-sdk"));
       g_ptr_array_add (args, g_strdup ("--sdk-dir=platform"));
+      if (self->var)
+        g_ptr_array_add (args, g_strdup_printf ("--var=%s", self->var));
```

Comment:
Actually, I don't think this is right.
The var either goes with the sdk or the platform. There is no way we can use the same for both. However, the way the platform is used is when we build a runtime and at the end we rebase all the changes from the "parent" sdk to the new sdk onto a newly checked out platform. When creating a platform like that I don't see how it ever makes sense to use a var runtime though.